src/main/kotlin/lotto/domain/vo/PurchaseAmount.kt

```diff
@@ -7,6 +7,8 @@ value class PurchaseAmount(val amount: Int) {
         require(amount % MIN_VALUE == 0) { "amount should be in units of $MIN_VALUE" }
     }
 
+    fun div(number: Int): Int = amount / number
```

Comment:
```suggestion
    operator fun div(number: Int): Int = amount / number
```

- 위 suggestion과 같은 식으로 사용한다면 amount / LOTTO_PRICE 처럼 연산자를 직접 활용할 수도 있습니다.
- PurchaseAmount가 로또의 구매금액을 나타내기 때문에 `puchaseAmount.countTicket()`등과 같은 함수가 있을수도 있다는 생각이 드네요 😄

Reply:
`operator` 너무 좋네요.. 

countTicket() 조금 더 명확한 것 같네요 반영하겠습니다~